app/services/quotes.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const logger = require('../logger');
+
+const getQuote = async (author, genre, count) => {
+  const url = 'https://quote-garden.herokuapp.com/api/v3/quotes/random';
+  let quote = {};
+  const params = {
+    author,
+    genre,
+    count
+  };
+
+  try {
+    const resService = await axios.get(url, { params });
+    quote = {
+      author: resService.data.data[0].quoteAuthor,
+      text: resService.data.data[0].quoteText
+    };
+  } catch (error) {
+    logger.error('Error in the service');
```

Comment:
Here you should show the error that the API throws you in the log and make use of the custom errors to throw an exception with a more generic message. Remember to combine the use of the `errors.js` and `middlewares/errors.js` files